lib/foreman_maintain/utils/service/systemd.rb

```diff
@@ -9,20 +9,15 @@ def initialize(name, priority, options = {})
       end
 
       def command(action, options = {})
-        do_wait = options.fetch(:wait, true) # wait for service to start
         all = @options.fetch(:all, false)
```

Comment:
eh, why is this using `@options` and not `options`?!

Reply:
answering myself, it wants the options of the object (as passed to `new`), not the ones to `command`…